### DISCUSSION TITLE
Muted mod not muting music

### DISCUSSION THREAD

**Discussion by wildmanlennox** [ppy/osu#21117] [Q&A]
### Type

Game behaviour

### Bug description

Toggling the mod while in song select only changes the volume to be slightly lower when enabled. 

### Screenshots or videos

https://user-images.githubusercontent.com/47006620/199929349-626bfb64-8e58-4d6b-94fb-54ebcf7a637d.mp4

### Version

2022.1101.0

### Logs

[database.log](https://github.com/ppy/osu/files/9936371/database.log)
[input.log](https://github.com/ppy/osu/files/9936372/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9936373/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9936374/network.log)
[performance.log](https://github.com/ppy/osu/files/9936375/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9936376/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9936377/updater.log)

**Comment by smoogipoo** [comment]
Fairly certain this was intentional at the time...

**Reply by peppy** [reply]
Check the mod settings? You likely aren't getting any kind of real combo.
